src/app/containers/MediaPlayer/helpers/placeholder.js

```diff
@@ -1,7 +1,7 @@
-const getPlaceholderSrc = src => {
+const getPlaceholderSrc = (src, imageWidth) => {
```

Comment:
I'd recommend setting a default for this

```suggestion
const getPlaceholderSrc = (src, imageWidth = 512) => {
```

Reply:
setting it to a default of 512 causes the image width to always render with a width of 512 irregardless of the devices screensize, keeping in mind we have to consider device pixel ratio

Reply:
NV. your suggestion works as well